SU-SWS/saa_victoria_subtheme
# Adapt 1679  event & news: SAA color styles

# ALMOST READY


# Summary
- Updating link and button color styles for News and Event pages, lists, etc.

# Review By (Date)
- 2/1/2021 morning, scheduled with Yvonne

# Urgency
- High - these pages need to be styled correctly for the Racial Justice site launch next Monday Feb 8th

# Steps to Test

1. Go to these pages:
 https://saauserguide-test.sites.stanford.edu/events
 https://saauserguide-test.sites.stanford.edu/news
 https://saauserguide-test.sites.stanford.edu/card-tests
 https://saauserguide-test.sites.stanford.edu/test-lists
2. Check that all instances of vivid red are the saa vivid red.
3. Check that every link hover is cardinalred with underline
4. Check that there is not underline until on hover on any of the links. (Except sidebar navigation top level links, i guess those always have underline)
5. Check that icon colors match the link colors correctly
6. check that topic links (both in cards and lists and for news and for events - 4 variations) are all normal font weight
7. Check code for improvements we could make in nesting, using @apply's etc. 
8.  Check that all buttons on these pages show as purple saa buttons.
9. Check that the left sidebar navigation links and their icons (on mobile and desktop) look correct with saa red on both ../events and ../news pages.
10. When left sidebar navigation is open, the links should be black but hover to cardinal red.
9. Check if anything doesn't look right on news or events pages. There is one block I think that we aren't styling yet, but let me know if you see anything looking bad.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
